server/lib/allowed-features.ts

```diff
@@ -17,14 +40,45 @@ export const OPT_OUT_FEATURE_FLAGS = {
 
 export const OPT_IN_FEATURE_FLAGS = {
   [FEATURE.CROSS_CURRENCY_MANUAL_TRANSACTIONS]: 'settings.features.crossCurrencyManualTransactions',
+  [FEATURE.COLLECTIVE_GOALS]: 'settings.collectivePage.showGoals',
+};
+
+const OPTIONAL_COLLECTIVE_FEATURES = {
+  [FEATURE.CONVERSATIONS]: 'conversations',
+  [FEATURE.UPDATES]: 'updates',
 };
 
 /**
  * Returns true if feature is allowed for this collective type, false otherwise.
  */
-export const isFeatureAllowedForCollectiveType = (collectiveType: keyof types, feature: FEATURE): boolean => {
+export const isFeatureAllowedForCollectiveType = (
+  collectiveType: types,
+  isHost: boolean,
+  feature: FEATURE,
+): boolean => {
   const allowedTypes = FeatureAllowedForTypes[feature];
-  return allowedTypes ? allowedTypes.includes(collectiveType) : true;
+  const allowedForType = allowedTypes ? allowedTypes.includes(collectiveType) : true;
+  const orgFeaturesOnlyAllowedForHostOrgs = [
```

Comment:
Maybe we can define this one as a constant outside of the function, as `FEATURES_ONLY_FOR_HOST_ORGS` (similar to what we do with `OPT_IN_FEATURE_FLAGS`)